controlplane/internal/kubevirt/mcs_proxy.go

```
@@ -0,0 +1,196 @@
+) (string, error) {
+	resolvedImage := GetMCSProxyImage(proxyImage)
+
+	deploy := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      clusterName + "-" + MCSProxyDeploymentName,
+			Namespace: namespace,
+		},
+	}
+
+	labels := map[string]string{
+		"app":                            clusterName + "-mcs-proxy",
```

golangci-lint (2.13.2)

[error] string `app` has 5 occurrences, make it a constant

(goconst)